css/style.css

```diff
@@ -0,0 +1,4 @@
+h1 {
+  color: blue;
+  /* stylelint-disable-next-line no-missing-end-of-source-newline */
```

Comment:
You did a great job adding inline comments and doing that review 💯 but there is still room for improvement, I noticed two things you can improve on all of your inline comments 👇🏽

- [ ] You should phrase the comments politely, as suggestions instead of orders.
For example:
  - This is a polite suggestion: `"I think it would be better to use semantic tags"`
  - This reads as an order: `"Use semantic tags"`

- [ ] You should always include an explanation, a "why", for each requested change. For example:
  - This is a well-explained request: `Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
  - This request lacks a "why": `Why don't you use an article tag here instead of a div?`

- [ ] I suggest you add emoji's to your comments so that the student understands the tone you are trying to communicate 😎.

NB: This should be taken into account for all of your inline comments and the required section of your Review Summary 🤓